VERSION

```diff
@@ -1,2 +1,2 @@
-elixir: 1.15.0
+elixir: 1.15.4
 erlang: 26.0.1
```

Comment:
It also looks like there is a new patch release for Erlang (https://github.com/erlang/otp/releases/tag/OTP-26.0.2), do you want to bump that at the same time?
```suggestion
erlang: 26.0.2
```